library/EasyPipeline/stack.yaml

```diff
@@ -6,92 +6,89 @@ packages:
 - ../Frontend
 - ../Core
 - ../Translate
+- ../Backends/Michelson
```

Comment:
Remove

Reply:
It's needed to make it compile

Reply:
This was part of the chnages needed to make `EasyPipeline` compile